src/Generators/Shared/RoslynExtensions.cs

```diff
@@ -0,0 +1,203 @@
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+#pragma warning disable CA1716
```

Comment:
What's this suppression?

Reply:
Identifiers should not match keywords. Added for consistency, as it's also found in other types in the M.Gen.Shared namespace.